plugins/modules/load_balancer_pool.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2023, Gaudenz Steinlin <gaudenz.steinlin@cloudscale.ch>
+# Copyright: (c) 2023, Kenneth Joss <kenneth.joss@cloudscale.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: load_balancer
+short_description: Manages load balancer pools on the cloudscale.ch IaaS service
+description:
+  - List, create, update, delete pools on the cloudscale.ch IaaS service.
+notes:
+  - If I(uuid) option is provided, it takes precedence over I(name) for pool selection. This allows to update the load balancer pool's name.
+  - If no I(uuid) option is provided, I(name) is used for pool selection. If more than one pool with this name exists, execution is aborted.
+author:
+  - Gaudenz Steinlin (@gaudenz)
+  - Kenneth Joss (@k-304)
+version_added: "2.3.0"
+options:
+  state:
+    description:
+      - State of the load balancer pool.
+    choices: [ present, absent ]
+    default: running
+    type: str
+  name:
+    description:
+      - Name of the load balancer pool.
+    type: str
+  load_balancer:
+    description:
+      - UUID of the load balancer for this pool.
+    type: str
+  algorithm:
+    description:
+      - The algorithm according to which the incoming traffic is distributed between the pool members.
+      - Currently the following algorithms are supported: I(round_robin), I(least_connections), I(source_ip).
+    type: str
+  protocol:
+    description:
+      - The protocol used for traffic between the load balancer and the pool members.
+      - Currently the following protocols are supported: I(tcp), I(proxy), I(proxyv2).
```

Comment:
Maybe just refer to the API documentation:
```suggestion
      - See the [API documentation](https://www.cloudscale.ch/en/api/v1#pool-protocols) for supported protocols.
```